README.md

```diff
@@ -4,12 +4,19 @@
 
 > While we are committed to following semver for this project, it's important to note that while we move through alpha and beta, minor version numbers can and will likely include breaking changes until we hit 1.0.0. After 1.0.0, we will be aligned with semver2.0.
 
-## What
+## Examples
+---
+### Using the components
+[README](https://github.com/Adaptive-Web-Community/Adaptive-Web-Components/tree/main/examples/use-adaptive-components/README.md)
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/Adaptive-Web-Community/Adaptive-Web-Components/tree/main/examples/use-adaptive-components)
 
+## What
+---
```

Comment:
Can we stick with standard GitHub formatting for headings? Does this add another rule line when rendered on GitHub? It does in VSCode.

Reply:
The web editor doesn't use GitHub's formatting apparently.